src/style.css

```diff
@@ -0,0 +1,118 @@
+
+body {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  font-family: roboto, sans-serif;
+  width: 200px;
+  height: 410px;
```

Comment:
[STYLES] - Don't hardcode exact height size. Add necessary paddings according to mockup and let content dictate the container size

checklist 8-th